components/GlobalNewsAndUpdates.js

```diff
@@ -6,7 +6,7 @@ import { useNewsAndUpdates } from './NewsAndUpdatesProvider';
 
 const GlobalNewsAndUpdates = () => {
   const { showNewsAndUpdates, setShowNewsAndUpdates } = useNewsAndUpdates();
-  return <NewsAndUpdatesModal show={showNewsAndUpdates} onClose={() => setShowNewsAndUpdates(false)} />;
+  return showNewsAndUpdates ? <NewsAndUpdatesModal onClose={() => setShowNewsAndUpdates(false)} /> : <React.Fragment />;
```

Comment:
We usually return `null` when a component has nothing to render. Is there a benefit in returning a fragment?

Reply:
TBH I'm not sure what I was thinking here, I'll return null. IIRC there was a time long, long ago when returning null was discouraged, so I guess in the flow of a very long session I might have had a flashback to then.